server/remote/bitbucket/bitbucket.go

```diff
@@ -301,6 +301,16 @@ func (c *config) Hook(ctx context.Context, req *http.Request) (*model.Repo, *mod
 	return parseHook(req)
 }
 
+// OrgMembership returns if user is member of organization and if user
+// is admin/owner in this organization.
+func (c *config) OrgMembership(ctx context.Context, u *model.User, owner string) (bool, bool, error) {
```

Comment:
do named returns or document values in desc

Reply:
having that in interface should be enaugh